lib/core/device.js

```diff
@@ -50,7 +52,9 @@ device._deviceComms = {
   'DexcomG4': serialDevice,
   'OneTouchMini': serialDevice,
   'AbbottFreeStyle': serialDevice,
+  'AsanteSNAP': serialDevice,
```

Comment:
We've removed the Asante drivers from the uploader, so it should not appear here.